_posts/2022-07-25-Building-CNF-Using-CNDP-Rust.md

```diff
@@ -0,0 +1,190 @@
+---
+layout: post
+title: Building Cloud Native Network Functions using Rust
+author: CNDP Team
+---
+
+
+### Rust for Networking and Cloud Native Development
```

Comment:
I think you don't need this header. The title is "Building Cloud Native Network Functions using Rust". I think the first paragraph should start here, without this extra header. When you render this page on the site, it looks weird because you basically have two titles in a row.

Reply:
I updated PR after removing the header

Reply:
thanks! It looks better now.